EternalModLoader/BlangJson.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+
+namespace EternalModLoader
+{
+    /// <summary>
+    /// BlangJson class for JSON deserialization
+    /// </summary>
+    public class BlangJson
+    {
+        /// <summary>
+        /// List of StringObjects for JSON deserialization
+        /// </summary>
+        public IList<StringObject> strings { get; set; }
+    }
+    
+    /// <summary>
+    /// StringObject class used by BlangJson class
+    /// </summary>
+    public class StringObject
+    {
+        /// <summary>
+        /// String identifier
+        /// </summary>
+        public string name { get; set; }
```

Comment:
Please follow C# coding conventions. See: https://docs.microsoft.com/es-es/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Please use PascalCase for properties.

If the JSON must be serialized, tell the serializer to use camelCase instead.